packages/bundle-source/src/index.js

```diff
@@ -192,6 +209,17 @@ async function bundleNestedEvaluateAndGetExports(
   });
   // console.log(output);
 
+  // Find the longest common prefix of all the source file names.
+  // We shorten the fileNames to make the bundle output deterministic.
```

Comment:
wasn't it "deterministic" before, just dependent on the host machine path?

Consider,
```suggestion
  // We shorten the fileNames to make the bundle output consistent between build environments.
```

Reply:
I like that phrasing